drivers/hmis/app/graphql/mutations/update_relationship_to_ho_h.rb

```diff
@@ -0,0 +1,71 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class UpdateRelationshipToHoH < BaseMutation
```

Comment:
Combined old UpdateEnrollment and SetHoHforEnrollment into one, since they are both only used for changing the relationship now. Simplifies things on the frontend too.